src/ontology/fbbt-edit.obo

```diff
@@ -121227,6 +121234,121 @@ relationship: part_of FBbt:00110844 ! presumptive ocellus
+creation_date: 2021-05-26T15:07:24Z
+
+[Term]
+id: FBbt:00058144
+name: adult plasmatocyte
+def: "Plasmatocyte of the adult. These cells all originate during embryonic and larval development. They do no divide in the adult and their numbers decrease slowly throughout adult life (Honti et al., 2014)." [FlyBase:FBrf0223115]
+subset: cur
+intersection_of: FBbt:00058145 ! plasmatocyte
+intersection_of: part_of FBbt:00003004 ! adult
+relationship: develops_from FBbt:00001685 ! embryonic/larval plasmatocyte
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-05-26T15:17:34Z
+
+[Term]
+id: FBbt:00058145
+name: plasmatocyte
+def: "Phagocyting hemocyte. These cells are responsible for the engulfment of small particles, microbes and apoptotic tissues debris (Honti et al., 2014). They express Nimrod C1 (Kurucz et al., 2007)." [FlyBase:FBrf0195027, FlyBase:FBrf0223115]
+subset: cur
+is_a: FBbt:00005063 ! hemocyte
+relationship: expresses http://flybase.org/reports/FBgn0259896 ! NimC1
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-05-26T15:18:53Z
+
+[Term]
+id: FBbt:00058146
+name: adult crystal cell
+def: "Crystal cell of the adult. Originating from the embryo and the larva, these cells persist in small numbers in the adult (Honti et al., 2014; Gold and Bruckner, 2015)." [FlyBase:FBrf0223115, FlyBase:FBrf0232226]
+subset: cur
+intersection_of: FBbt:00058147 ! crystal cell
+intersection_of: part_of FBbt:00003004 ! adult
+relationship: develops_from FBbt:00001690 ! embryonic/larval crystal cell
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-05-26T15:20:03Z
+
+[Term]
+id: FBbt:00058147
+name: crystal cell
+def: "Melanizing hemocyte. It has roughly the same size as a plasmatocyte but is distinguishable by its crystal inclusions of phenoloxidases (encoded by PPO1 and PPO2) that are necessary for the synthesis of melanin as part of the antimicrobial immune reaction (Wood and Jacinto, 2007; Honti et al., 2014; Cattenoz et al., 2020)." [FlyBase:FBrf0202336, FlyBase:FBrf0223115, FlyBase:FBrf0245988]
+subset: cur
+is_a: FBbt:00005063 ! hemocyte
+relationship: expresses http://flybase.org/reports/FBgn0033367 ! PPO2
+relationship: expresses http://flybase.org/reports/FBgn0283437 ! PPO1
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-05-26T15:20:21Z
+
+[Term]
+id: FBbt:00058148
+name: sessile hemocyte
+def: "Hemocyte that is bound to a tissue of the larva or adult, instead of circulating in the hemolymph (Honti et al., 2014)." [FlyBase:FBrf0223115]
+subset: cur
+synonym: "resident hemocyte" EXACT [FlyBase:FBrf0244363]
+is_a: FBbt:00005063 ! hemocyte
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-05-26T17:41:13Z
+
+[Term]
+id: FBbt:00058149
+name: circulating hemocyte
+def: "Hemocyte that is freely circulating in the hemolymph (Honti et al., 2014)." [FlyBase:FBrf0223115]
+subset: cur
+is_a: FBbt:00005063 ! hemocyte
```

Comment:
Not entirely I think, as the `part of some hemolymph` of the logical definition may not be enough to fully convey the idea of "freely circulating". But I will add something like "instead of being bound to tissues" to make that even clearer (to mirror the definition of "sessile hemocyte").

Reply:
I mean line 131315 'is_a: FBbt:00005063 ! hemocyte' is redundant with line 121316 'intersection_of: FBbt:00005063 ! hemocyte'

Reply:
Doh! My bad, I'll fix that right away.